docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectAnomalyBySrCnnBatchPrediction.cs

```diff
@@ -18,88 +19,80 @@ public static void Example()
             var data = new List<TimeSeriesData>();
             for (int index = 0; index < 20; index++)
             {
-                data.Add(new TimeSeriesData(5));
+                data.Add(new TimeSeriesData { Value = 5 });
             }
-            data.Add(new TimeSeriesData(10));
+            data.Add(new TimeSeriesData { Value = 10 });
             for (int index = 0; index < 5; index++)
             {
-                data.Add(new TimeSeriesData(5));
+                data.Add(new TimeSeriesData { Value = 5 });
             }
 
             // Convert data to IDataView.
             var dataView = ml.Data.LoadFromEnumerable(data);
 
-            // Setup the estimator arguments
+            // Setup the detection arguments
             string outputColumnName = nameof(SrCnnAnomalyDetection.Prediction);
             string inputColumnName = nameof(TimeSeriesData.Value);
 
-            // The transformed data.
-            var transformedData = ml.Transforms.DetectAnomalyBySrCnn(
```

Comment:
>DetectAnomalyBySrCnn [](start = 48, length = 20)

This old API still exists. Shouldn't we add a new sample instead of changing this one to the new API? Or, alternatively, showing both use cases in this sample.

Reply:
The streaming API's output and entire API's output are different due to the implementing strategy, so it may not be proper to put them in one sample. If we want to preserve the old sample, how about name the new class "SrCnnEntireAnomalyDetector", and create a new related sample for the entire API?

---
In reply to: [426446728](https://github.com/dotnet/machinelearning/pull/5135#discussion_r426446728) [](ancestors = 426446728)

Reply:
Yes, Please add a new sample (or at the very least, an option to run the same code with either API)

---
In reply to: [426703738](https://github.com/dotnet/machinelearning/pull/5135#discussion_r426703738) [](ancestors = 426703738,426446728)